static_src/util/cf_api.js

```diff
@@ -237,6 +239,10 @@ export default {
                           appActions.receivedApp);
   },
 
+  fetchAppStatus(appGuid) {
```

Comment:
I created a new method because the fetch app method will call the app received action which will update the UI which I don't want to have happen. I really just want a promise that gets the data and returns it. But there might be another way to re-use the existing fetchApp method rather then creating another one like this.